mlflow/pipelines/pipeline.py

```diff
@@ -41,6 +41,7 @@ def __init__(self, pipeline_root_path: str, profile: str) -> None:
                         pipeline.yaml to generate the configuration to run the pipeline.
         """
         self._pipeline_root_path = pipeline_root_path
+        self._initial_config_step = ""
```

Comment:
`_initial_config_step` -> `_run_args`, and record all args provided to `Pipeline.run()`. This will come in handy in future iterations we plan to add more args to the API.